src/DataTypes/ErrorCode.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * Contains CBS\SmarterU\DataTypes\ErrorCode
+ *
+ * @author Tom Egan <tom.egan@thecoresolution.com>
+ * @copyright $year$ Core Business Solutions
+ * @license Proprietary
```

Comment:
I propose we put things like this on the repository [Issues](https://github.com/cbs-software/smarteru-php-client/issues) since it is intended to be open source and not depend on our internal project management boards.